src/System.Private.ServiceModel/src/System/ServiceModel/Channels/SslStreamSecurityUpgradeProvider.cs

```diff
@@ -303,139 +284,6 @@ protected override void OnEndOpen(IAsyncResult result)
         }
     }
 
-    internal class SslStreamSecurityUpgradeAcceptor : StreamSecurityUpgradeAcceptorBase
```

Comment:
Not blocking -- it would be helpful if you could comment in this PR why some of these types are being deleted.  It's a little difficult to grock the whole picture from a PR.